packages/modules/src/modules/banners/common/PaymentCards.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { css, SerializedStyles } from '@emotion/react';
+import { space } from '@guardian/src-foundations';
+import { from } from '@guardian/src-foundations/mq';
+
+const paymentMethods = css`
+    display: block;
+    height: 1.1rem;
+    width: auto;
+    margin-top: ${space[2]}px;
+    margin-left: ${space[1]}px;
+
+    ${from.tablet} {
+        margin-left: ${space[2]}px;
+        height: 1.25rem;
+    }
+`;
+
+type PaymentCardProps = {
+    cssOverrides?: SerializedStyles;
+};
+
+export const PaymentCards: React.FC<PaymentCardProps> = ({ cssOverrides = [] }) => {
```

Comment:
I think we've typed components in this repo a few different ways but I might suggest our typical approach in support frontend here:

```suggestion
export const PaymentCards = ({ cssOverrides }: PaymentCardProps): JSX.Element => {
```

Reply:
Ah, was not sure which way was more modern. All done.